docs/userguides/trustedactivities.md

```diff
@@ -0,0 +1,52 @@
+# Trust Settings
+
+Use py42 to quickly and easily manage trusted activities and resources, including domains and Slack workspaces.
+
+## Create, View or Modify Trusted Activities
+
+To get started create a new trusted activity resource.
+
+```python
+from py42.constants import TrustedActivityType
+
+response = sdk.trustedactivities.create(TrustedActivityType.DOMAIN, "test-domain.com")
+```
+Trusted activities can be of type `DOMAIN` or `SLACK`.
```

Comment:
Instead of enumerating all activities here, we should add all the newly imported enum classes to the `docs/methoddocs/constants.md` file, then link `TrustedActivityType` here, so if we add stuff to the enum later on the docs stay up-to-date.

Reply:
Thats a good idea, I'll do that for the ones listed under the methods as well.